staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -453,6 +460,36 @@ type DeleteOptions struct {
 	// foreground.
 	// +optional
 	PropagationPolicy *DeletionPropagation `json:"propagationPolicy,omitempty" protobuf:"varint,4,opt,name=propagationPolicy"`
+
+	// DryRun decides if the request should be completed or not.
```

Comment:
make the documentation match the serialized field name: `dryRun`

the documentation for this field should:
* indicate the guarantee of setting the field (the mutation will not be persisted)
* indicate what the recognized values mean ("All" means ____)
* indicate what will be done with unrecognized values